### ISSUE TITLE
I18n support I want contribute with string translation 

### ISSUE THREAD

**Issue by johnfelipe** [hellomuthu23/planning-poker#77]

**Comment by hellomuthu23** [comment]
Good idea, I am not sure if the code base is ready to support I18n yet. If you got a PR, I can support it

**Comment by johnfelipe** [comment]
If you have strings I can translate to Spanish and french

**Comment by vnscosta** [comment]
Hey, I can contribute to this one, @hellomuthu23 can you assign it to me?

**Comment by hellomuthu23** [comment]
@vnscosta - thanks a lot, assigned it to you now

**Comment by hellomuthu23** [comment]
Translation feature implemented now. Closing this issue
